arduino_controller:
  ros__parameters:
    serial_port: "/dev/ttyUSB0"
    steering_axis: 2
    throttle_axis: 3